tests/integration/data_types/test_examples_10.py

```diff
@@ -98,24 +102,46 @@ def test_header_with_mutiple_examples(api):
 
     ex, = h.examples.filter_by(name="simple")
     assert ex.description is None
-    assert ex.display_name.raw == "simple"
+    assert ex.display_name == "simple"
     assert ex.strict is True
     assert ex.value == "42"
 
     ex, = h.examples.filter_by(name="typical")
     assert ex.description.raw == "This is what we expect."
-    assert ex.display_name.raw == "Typical Value"
+    assert ex.display_name == "Typical Value"
     assert ex.strict is True
     assert ex.value == "typical"
 
     ex, = h.examples.filter_by(name="special")
     assert ex.description.raw == "No one expects the ..."
-    assert ex.display_name.raw == "Surprise Visit"
+    assert ex.display_name == "Surprise Visit"
     assert ex.strict is True
     assert ex.value == "Spanish Inqusition!"
 
     ex, = h.examples.filter_by(name="broken")
     assert ex.description.raw == "Send this for a 500"
-    assert ex.display_name.raw == "DON'T DO THIS"
+    assert ex.display_name == "DON'T DO THIS"
     assert ex.strict is False
     assert ex.value == "breakfast"
+
+
+def test_disallows_example_with_examples():
+    ramlfile = os.path.join(RAML_10, "broken-example-with-examples.raml")
+    loaded_raml = load_file(ramlfile)
+    conffile = os.path.join(RAML_10, "test-config.ini")
+    config = setup_config(conffile)
+    try:
+        parse_raml(loaded_raml, config)
+    except errors.InvalidRAMLStructureError as e:
+        assert str(e) == 'example and examples cannot co-exist'
```

Comment:
The way to assert that an exception has been raised is not with a try/except but with a `with pytest.raises`:

```python
with pytest.raises(errors.InvalidRAMLStructureError) as e:
    parse_raml(loaded_raml, config)

msg = # error message
assert msg in e.value.args[0]
```